querybook/webapp/components/DataDocDAGExporter/DataDocDAGExporter.tsx

```diff
@@ -55,60 +58,73 @@ export const DataDocDAGExporter: React.FunctionComponent<IProps> = ({
         [docId, nodes, edges, onSave]
     );
 
+    const DAGExporterContextState = useMemo(() => {
+        const isEngineSupported = (engineId: number) =>
```

Comment:
yep, `currentExporter.engines` is already the list of supported engine ids, while `currentExporter` could be null when the component is first loaded. we'd like to have the error icon of unsupported engine to show up after the current exporter is loaded instead of showing it from begining.